build.gradle

```diff
@@ -117,7 +117,7 @@ dependencies {
     compile("ch.qos.logback.contrib:logback-json-classic:0.1.5")
     compile("ch.qos.logback.contrib:logback-jackson:0.1.5")
     compile("net.logstash.logback:logstash-logback-encoder:5.2")
-    compile("com.checkmarx:cx-config-provider:1.0.8")
+    compile("com.checkmarx:cx-config-provider:1.0.9")
```

Comment:
can we add a separate property to define cx-provider version, like we do with cx-sdk?

Reply:
Sure.